docs/how-to-start.md

```diff
@@ -37,6 +37,9 @@ Or pipe output of the make target:
 ```bash
 $ make image 2>&1 | sed -e "s|/src|${PWD}/collector/|g"
 ```
+*NOTE*: To build a multi-arch image, use the `ma-image` target instead which
+uses `docker buildx` to build and push the image to registry in a single step.
```

Comment:
Why do we need to push directly? Why can't we just build the image in our makefiles then push them in CI the way we do with the existing builds?

Reply:
We need to push directly because docker does not allow loading the multi-arch manifest into it's engine (`docker image ls`) at the current time. To work around this, buildx pushes the multi-arch manifest directly to a registry.